coredb-operator/src/statefulset.rs

```diff
@@ -26,6 +27,61 @@ pub async fn reconcile_sts(cdb: &CoreDB, ctx: Arc<Context>) -> Result<(), Error>
     labels.insert("app".to_owned(), "coredb".to_owned());
     pvc_requests.insert("storage".to_string(), Quantity("8Gi".to_string()));
 
+    let postgres_env = Some(vec![EnvVar {
+        name: "POSTGRES_PASSWORD".to_owned(),
+        value: Some("password".to_owned()),
+        value_from: None,
+    }]);
+
+    let postgres_volume_mounts = Some(vec![VolumeMount {
+        name: "data".to_owned(),
+        mount_path: "/var/lib/postgresql/data".to_owned(),
+        ..VolumeMount::default()
+    }]);
+
+    // This container for running postgresql
+    let postgres_container = Container {
+        env: postgres_env.clone(),
+        security_context: Some(SecurityContext {
+            run_as_user: Some(999),
```

Comment:
Would we ever want to change this? If yes, we should add a spec value like `UID`.

Reply:
Good call, added that plus a test for it